win32ctypes/tests/test_win32api.py

```diff
@@ -162,11 +162,11 @@ def test_update_resource(self):
         # when
         handle = module.BeginUpdateResource(filename, False)
         try:
-            module.UpdateResource(
+            self.assertEqual(module.UpdateResource(
                 handle, resource_type, resource_name, resource[:-2],
-                resource_language)
+                language=resource_language), 1)
         finally:
-            module.EndUpdateResource(handle, False)
+             self.assertEqual(module.EndUpdateResource(handle, False), 1)
```

Comment:
This assertion should probably be moved to the `test_begin_and_end_update_resource`